modules/scrabble.js

```diff
@@ -44,10 +46,19 @@ function scoreLetter (letter, str) {
     }
   }
 }
+function scrabbleNotate (str) {
+  let word = str.toLowerCase()
+  for (const letter of [...new Set(word)]) {
+    for (let i = 0; i < bag[letter]['count']; i++) {
+      word = word.replace(letter, letter.toUpperCase())
+    }
+  }
+  return word + (sowpods.verify(word) ? '' : '*')
+}
 function getPunctuation (bot, score) {
   const minScore = bot.config.get('scrabble.minScore')
   const exclThreshold = bot.config.get('scrabble.exclamationThreshold')
-  const numberOfMarks = Math.floor((score - minScore) / exclThreshold)
+  const numberOfMarks = Math.max(Math.floor((score - minScore) / exclThreshold), 0)
```

Comment:
What does `.max()` do here? If `numberOfMarks` is falsy the line below will handle it

Reply:
The next line tries to `.repeat` that many times. If the value is negative, that doesn't work.